src/utilities/spacing/spacing.scss

```diff
@@ -41,11 +47,17 @@
     }
   }
 
+  /**
+    * 1. Ideally we could use the margin-inline shorthand, but it is not as well
+    *    supported in browsers.
+    */
   &-inline-#{$name} {
     @if $property == margin {
-      margin-inline: $value;
+      margin-inline-end: $value; /* 1 */
+      margin-inline-start: $value;
     } @else {
-      padding-inline: $value;
+      padding-inline-end: $value; /* 1 */
+      padding-inline-start: $value;
```

Comment:
I don't think the number is necessary at all, since the comment is applying to the whole block of styles.